### ISSUE TITLE
Unable to update agent details

### ISSUE THREAD

**Issue by iHunt101** [osTicket/osTicket#4761]
### Description

Updating agent details show error

> Email already in use by another agent

> Username already in use

### Steps to Reproduce
1. Create new Agent
2. Update details (aside from email and username)
3. Save changes

**Expected behavior:** Should allow me to update permissions, access and teams

**Actual behavior:** Error show that username and email already exist

### Versions
osTicket v1.11 (7b1eee8)
MySQL Version | 5.5.48
PHP Version | 5.6.21
Google Chrome Version 72.0.3626.119
Winsows Server 2008 IIS

**Comment by JediKev** [comment]
@iHunt101 

I'm totally not seeing the same thing...maybe the email and username does already exist?
![update_agent_v111](https://user-images.githubusercontent.com/11823401/53503150-27831f00-3a75-11e9-8802-1d88bf58a5db.gif)

I tested using exact setup minus IIS, I'm using Apache.

Cheers.

**Comment by iHunt101** [comment]
Did you try to make Team and then update agent details by adding her in team. Then change her Team then save?

**Comment by JediKev** [comment]
@iHunt101 

No, you never mentioned that...that's why it's good to fill out the entire form with _exact_ steps to replicate...

Will try again..

Cheers.

**Comment by JediKev** [comment]
@iHunt101 

Just tried creating a new team, added the new agent to the team, updated phone number + changed team, saved changes and still cannot reproduce your issue. Any more steps to try? Did I do it wrong?

Cheers.

**Comment by iHunt101** [comment]
Sorry for that.. Here is what i did.
1. Create new Agent & New Team
2. Update Agent phone number
3. Click "Team Tab"
4. Add current Agent to any Team then Save changes
5. Edit details of the Agent
6. Change the Team where it belong before
7. Save changes
8. Error shows that username and email already in use. But i never change any of it.

Thanks!
